common/persistence/cassandra/cassandraHistoryPersistence.go

```diff
@@ -22,46 +22,29 @@ package cassandra
 
 import (
 	"fmt"
-	"sort"
```

Comment:
I will fix import later. Also this folder will be moved to "nosql" pkg later. Right now I keep in this folder so that the code review doesn't look crazy :D